### DISCUSSION TITLE
BottomSheets这个组件已经用了zustand,为什么还用provider传递一下?

### DISCUSSION THREAD

**Discussion by fxwing** [actifyjs/actify#12] [Q&A] [answered]
已经上了zustand而且在react中为什么使用createStore和useStore?
这两个hook是在非react环境用的
直接用create api不更简单吗?

**Comment by lerte** [comment, marked as answer]
其实不应该使用zustand的，直接使用react context就好了，后面有可能会移除zustand，当时我为了用zustand而用zustand，因为需要使用props初始化state，所以需要provider。https://docs.pmnd.rs/zustand/guides/initialize-state-with-props

**Comment by fxwing** [comment]
> BottomSheets这个组件已经用了zustand,为什么还用provider传递一下?

ok  了解了
